Introduction to Tecton/Tecton Tutorial: Introduction.py

```diff
@@ -62,8 +62,8 @@
 start_time = datetime.utcnow()-timedelta(days=14)
 end_time = datetime.utcnow()
 
-features = fv.run(start_time=start_time, end_time=end_time).to_pandas()
-features.sort_values(by="is_fraud_mean_1d_1d", ascending=False).head(5)
+features = fv.run(start_time=start_time, end_time=end_time).to_spark()
+display(features.orderBy("is_fraud_mean_1d_1d", ascending=False).limit(5))
```

Comment:
idk if we'd prefer to order by or filter non-zeroes? limit is not necessary when using display, might be nice to show more